airbyte-protocol/protocol-models/src/test/java/io/airbyte/protocol/models/CatalogHelpersTest.java

```diff
@@ -73,9 +77,35 @@ void testGetFieldNames() throws IOException {
     final JsonNode node = Jsons.deserialize(MoreResources.readResource("valid_schema.json"));
     final Set<String> actualFieldNames = CatalogHelpers.getAllFieldNames(node);
     final Set<String> expectedFieldNames =
-        ImmutableSet.of("date", "CAD", "HKD", "ISK", "PHP", "DKK", "HUF", "文", "somekey", "something", "nestedkey");
+        ImmutableSet.of("date", "CAD", "HKD", "ISK", "PHP", "DKK", "HUF", "文", "somekey", "something", "nestedkey", "something2");
```

Comment:
Nit:  Can use `Set.of()` instead of the Guava `ImmutableSet` to achieve the same thing.